classes/State.php

```diff
@@ -142,7 +142,7 @@ public function initDefault(Upgrader $upgrader, $prodRootDir, $version)
 
         // installedLanguagesIso is used to merge translations files
         $installedLanguagesIso = array_map(
-            function ($v) { return $v['iso_code']; },
+            static function ($v) { return $v['iso_code']; },
```

Comment:
Since php 5.4 and this module need to be avaiable for php 5.2 due to PrestaShop 1.6.